### ISSUE TITLE
I can't create a build with Geospatial API for Android

### ISSUE THREAD

**Issue by victorgarri** [google-ar/arcore-unity-extensions#222]
With Unity 2022.3.17f1, AR Foundation 5.1.5 and ARCore extension 1.46, when creating a build it gets stuck and does not finish. I have waited more than 30 minutes and the build is not created. What could be happening?

**Comment by codedotart** [comment]
Yes I am having the same issue. When I open "Display Libraries". It doesn't show anything.
